google/datalab/contrib/pipeline/composer/setup.sh

```diff
@@ -0,0 +1,25 @@
+ZONE=us-central1
+PROJECT=$1
+EMAIL=$2
+ZONE=$3
+ENVIRONMENT=$4
+
+
+gcloud config set project $PROJECT
+gcloud auth login --activate $EMAIL
+sudo gcloud components repositories add https://storage.googleapis.com/composer-trusted-tester/components-2.json
+
+#gcloud components update -q
+sudo apt-get update
+sudo apt-get --only-upgrade install kubectl google-cloud-sdk google-cloud-sdk-datastore-emulator \
+google-cloud-sdk-pubsub-emulator google-cloud-sdk-app-engine-go google-cloud-sdk-app-engine-java \
+google-cloud-sdk-app-engine-python google-cloud-sdk-cbt google-cloud-sdk-bigtable-emulator \
+google-cloud-sdk-datalab
```

Comment:
We can put apt-get first, and then composer gcloud installation, and finally composer related configs and environment creation.